src/app/mod.rs

```diff
@@ -383,9 +384,10 @@ impl Surreal {
 	/// ```
 	pub async fn create(&self, resource: String, data: JsValue) -> Result<JsValue, Error> {
 		let resource = Resource::from(resource);
-		let response = match from_value::<Option<Json>>(data)? {
-			Some(data) => self.db.create(resource).content(data).await?,
-			None => self.db.create(resource).await?,
+		let data = json(&from_value::<Json>(data)?.to_string()).unwrap_or(Value::None);
+		let response = match data.is_object() {
+			true => self.db.create(resource).content(data).await?,
+			false => self.db.create(resource).await?,
 		};
```

Comment:
Let's please forward non-objects as well. As long as `data` is not null or undefined we should forward it.

Reply:
I believe we should only not forward undefined. undefined is no value, null is an empty value, but a value.